rts/Sim/Units/UnitDef.cpp

```diff
@@ -388,45 +451,69 @@ UnitDef::UnitDef(const LuaTable& udTable, const std::string& unitName, int id)
 
+	/*** @field UnitDef.fullHealthFactory boolean? (Default: `false`) */
 	fullHealthFactory = udTable.GetBool("fullHealthFactory", false);
+	/*** @field UnitDef.factoryHeadingTakeoff boolean? (Default: `true`) */
 	factoryHeadingTakeoff = udTable.GetBool("factoryHeadingTakeoff", true);
 
+	/*** @field UnitDef.upright boolean? (Default: `false`) */
 	upright = udTable.GetBool("upright", false);
+	/*** @field UnitDef.blocking boolean? (Default: `true`) */
 	collidable = udTable.GetBool("blocking", true);
+	/*** @field UnitDef.collider boolean? (Default: `true`) */
 	collide = udTable.GetBool("collide", true);
 
+	/*** @field UnitDef.maxSlope number? Maximum slope in degrees. In range `[0, 89.0]` */
 	const float maxSlopeDeg = std::clamp(udTable.GetFloat("maxSlope", 0.0f), 0.0f, 89.0f);
 	const float maxSlopeRad = maxSlopeDeg * math::DEG_TO_RAD;
 
 	// FIXME: kill the magic constant
 	maxHeightDif = 40.0f * math::tanf(maxSlopeRad);
 
+	/*** @field UnitDef.minWaterDepth number? (Default: `-10e6`) */
 	minWaterDepth = udTable.GetFloat("minWaterDepth", -10e6f);
+	/*** @field UnitDef.maxWaterDepth number? (Default: `+10e6`) */
 	maxWaterDepth = udTable.GetFloat("maxWaterDepth", +10e6f);
+	/*** @field UnitDef.waterline number? (Default: `0.0`) */
 	waterline = udTable.GetFloat("waterline", 0.0f);
+	/*** @field UnitDef.minCollisionSpeed number? (Default: `1.0`) */
 	minCollisionSpeed = udTable.GetFloat("minCollisionSpeed", 1.0f);
+	/*** @field UnitDef.slideTolerance number? (Default: `0.0`) */
 	slideTolerance = udTable.GetFloat("slideTolerance", 0.0f); // disabled
+	/*** @field UnitDef.rollingResistanceCoefficient number? (Default: `0.05`) */
 	rollingResistanceCoefficient = udTable.GetFloat("rollingResistanceCoefficient", 0.05f);
+	/*** @field UnitDef.groundFrictionCoefficient number? (Default: `0.01`) */
 	groundFrictionCoefficient = udTable.GetFloat("groundFrictionCoefficient", 0.01f);
+	/*** @field UnitDef.atmosphericDragCoefficient number? (Default: `1.0`) */
 	atmosphericDragCoefficient = udTable.GetFloat("atmosphericDragCoefficient", 1.0f);
+	/*** @field UnitDef.pushResistant boolean? (Default: `false`) */
 	pushResistant = udTable.GetBool("pushResistant", false);
+	/*** @field UnitDef.selfDestructCountdown integer? (Default: `5`) */
 	selfDCountdown = udTable.GetInt("selfDestructCountdown", 5);
 
 	/* Note that the legacy unit is elmo/frame
 	 * whereas the modern one is elmo/second */
+
+	/*** @field UnitDef.speed maxVelocity? (Default: `0`) Maximum speed in elmo/second. */
 	const decltype(speed) speedLegacy = math::fabs(udTable.GetFloat("maxVelocity", 0.0f) * GAME_SPEED);
 	speed = udTable.GetFloat("speed", speedLegacy);
 	if (speed < 0.0f)
 		throw content_error(unitName + ".speed < 0");
 
+	/*** @field UnitDef.speed maxReverseVelocity? (Default: `0`) Maximum reverse speed in elmo/second. */
 	const decltype(rSpeed) rSpeedLegacy = math::fabs(udTable.GetFloat("maxReverseVelocity", 0.0f) * GAME_SPEED);
```

Comment:
I think this one might be in frames, it is multiplied by `GAME_SPEED`

Reply:
`maxReverseVelocity` is in elmo/frame and is deprecated.

`rSpeed` (note the R btw, the line added by the PR says just "speed") is the modern spelling and is in elmo/second.